README.markdown

```diff
@@ -2,6 +2,9 @@ Provide markdown conversion (based on [Michel Fortin work](https://github.com/mi
 
 [![Build Status](https://secure.travis-ci.org/KnpLabs/KnpMarkdownBundle.svg)](http://travis-ci.org/KnpLabs/KnpMarkdownBundle)
 
+![ci.yml](https://github.com/tacman/KnpMarkdownBundle/actions/workflows/ci.yml/badge.svg)
+![php.yml](https://github.com/tacman/KnpMarkdownBundle/actions/workflows/php.yml/badge.svg)
```

Comment:
I think we need to wrap these icons with a link to the latest builds as it's done for TravisCI above

Reply:
agreed, is there a way to substitute {reponame} -- otherwise in my branch it was getting the icons from the main repository.  There's likely some better way to do this that I just don't know.